### PR TITLE
DO NOT MERGE Python 3 test

### PR DESCRIPTION
**Pull request by millerdev**
For python 3 tests

Might need to restart the build after the [docker hub build](https://hub.docker.com/r/dimagi/commcarehq_py3/builds/) is finished. It's still queued as I'm writing this.

@nickpell

### CODE REVIEW

**Comment by gcapalbo** [comment]
@nickpell fyi we can't completely delete the touchforms submodule until all the commcare-cloud references are removed. everything is mentioned in https://trello.com/c/jGzPtKKR/120-remove-references-to-touchforms

**Comment by nickpell** [comment]
@gcapalbo 👍 I'm treating this as an experimental branch, not to be merged

**Comment by nickpell** [comment]
Note to anyone getting notifications on this thread: you're welcome to follow along, but you might consider muting this thread to avoid getting spammed.  I expect to have this PR open until py3 tests are passing on master and don't expect anyone to have to review this PR.

File: corehq/motech/repeaters/tests/test_repeater.py [hunk 1]
```diff
@@ -104,6 +105,7 @@ def repeat_records(cls, domain_name):
         return RepeatRecord.all(domain=domain_name, due_before=later)
 
 
+@skip
```
Comment (kaapstorm) [inline corehq/motech/repeaters/tests/test_repeater.py]: Sorry. I'll get on this as soon as I have a chance.

**Pull request closed without merging**